### DISCUSSION TITLE
BSCE

### DISCUSSION THREAD

**Discussion by Boganabi** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#25] [General]
Discuss things related to BSCE here!

**Comment by vamsikrishnabandaru3** [comment]
**How to Install Burp Suite Community Edition on Your Laptop**

**Step 1:** Download Burp Suite Community Edition
- Go to the official Burp Suite website: https://portswigger.net/burp/communitydownload
- Select the appropriate version based on your operating system (Windows, macOS, or Linux).
   - Windows: .exe installer
   - macOS: .dmg installer
   - Linux: .sh installer
- Click Download to start the download process.

**Step 2:** Install Burp Suite on Your Laptop
- For Windows Users
   - Locate the downloaded .exe file
   - Double-click the file to launch the installation wizard
   - Follow the on-screen prompts
   - Click Next to agree to the license terms
   - Select the installation folder (default is recommended)
   - Click Install to begin the installation
   - Once installed, click Finish and launch Burp Suite

- For macOS Users
   - Locate the .dmg file in your downloads folder
   - Double-click it to open the installer
   - Drag the Burp Suite icon to the Applications folder
   - Open Applications, and double-click Burp Suite to launch it

- For Linux Users
   - Open the terminal and navigate to the folder where the .sh file was downloaded.
   - Run the following command to give the installer executable permissions
      - `chmod +x burpsuite_community_linux_v*.sh`
   - Run the installer: `./burpsuite_community_linux_v*.sh`

Follow the installation instructions in the terminal
Once completed, launch Burp Suite by typing burpsuite in the terminal
